examples/supabase-worker/.env.example

```
@@ -0,0 +1,10 @@
+# Copy to .env.local before running `supabase functions serve`.
+# Get these from your CipherStash workspace dashboard.
+
+CS_WORKSPACE_CRN=
+CS_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CS_CLIENT_ID key should go before the CS_WORKSPACE_CRN key

(UnorderedKey)